zerver/tests/tests.py

```diff
@@ -74,6 +74,9 @@ def find_dict(lst, k, v):
             return dct
     raise Exception('Cannot find element in list where key %s == %s' % (k, v))
 
+# same as in test_uploads.py
+TEST_AVATAR_DIR = os.path.join(os.path.dirname(__file__), 'images')
```

Comment:
Maybe move this to be defines in test_helpers.py?  Seems better than defining in two places.